rest-api/offline/metrics_export.py

```diff
@@ -30,8 +30,8 @@
     WHERE bo.participant_id = p.participant_id) first_order_date,
   (SELECT ISODATE[MIN(bs.confirmed)] FROM biobank_stored_sample bs
     WHERE bs.biobank_id = p.biobank_id) first_samples_arrived_date,
-  (SELECT ISODATE[MIN(pm.created)] FROM physical_measurements pm
-    WHERE pm.participant_id = p.participant_id) first_physical_measurements_date,
+  (SELECT ISODATE[MIN(pm.finalized)] FROM physical_measurements pm
+    WHERE pm.participant_id = p.participant_id and pm.finalized is not null) final_physical_measurements_date,
```

Comment:
Note that after the backfill is done, all physical measurements should have finalized timestamps, so this is not null won't be needed... but it doesn't hurt anything.

Reply:
ok